pkg/kv/kvserver/replica_app_batch.go

```diff
@@ -254,6 +261,9 @@ func (b *replicaAppBatch) runPostAddTriggersReplicaOnly(
 	// expected to ensure that no rangefeeds are currently active across such
 	// spans, but as a safeguard we disconnect the overlapping rangefeeds
 	// with a non-retriable error anyway.
+	//
+	// The are no rangefeeds in standalone mode, so we don't have to do anything
+	// for this on appBatch.
```

Comment:
Why not?

If the standalone application is going to be used for restart recovery / replaying commands, wouldn't we want to recover all sorts of mechanisms like rangefeeds?

Or the idea is that standalone application is only about recovering storage to the correct state?

Reply:
A rangefeed is a subscription, so there has to be a consumer - if we're just replaying the log "offline" there is no consumer.
For testing, it might still be worth integrating some rangefeed-related functionality in standalone mode, though I'm not doing that here as it would be a detour.

Reply:
Gotcha. Is the subsription reflected in storage? (e.g. is there some "watermark" state for a subscriber, so that things can be recovered etc?)

Reply:
No, the subscription is ephemeral from the point of view of storage. The way it works is that the consumer will come with a low water mark, do a catch-up scan, and then consume the raft log "directly" (i.e. the steady state). It will also receive resolved timestamps (which will raise the low water mark for future catch-up scans should they become necessary).